docs/reference/release-notes/8.9.1.asciidoc

```diff
@@ -3,6 +3,25 @@
 
 Also see <<breaking-changes-8.9,Breaking changes in 8.9>>.
 
+[float]
+[[security-updates-8.9.1]]
```

Comment:
Shouldn't this be in the release notes for `8.9.2` instead of `8.9.1` ?
It's been a while since we've included a security issue in the release notes (which surprises me, but that's a separate discussion), but the most recent relevant one I could find was for [7.13.4 and we included the security info in the release that fixes the problem](https://www.elastic.co/guide/en/elasticsearch/reference/7.13/release-notes-7.13.4.html).

Reply:
Thanks @tvernum. I've relocated them to `8.9.2` with [`6db98bb` (#99949)](https://github.com/elastic/elasticsearch/pull/99949/commits/6db98bb014c39cf729d406669aff1d1d72c8e8f3).

> the most recent relevant one I could find was for [7.13.4 and we included the security info in the release that fixes the problem](https://www.elastic.co/guide/en/elasticsearch/reference/7.13/release-notes-7.13.4.html).

I'll note this in the process for docs release coordinators going forward. Thanks for checking on this!